applications/API/models.py

```
@@ -0,0 +1,626 @@
+    "CACKLE",
+]
+
+
+class Base64DecoderException(Exception):
+    """Custom exception for base36 decoding errors."""
+
+class Base64EncoderException(Exception):
+    """Custom exception for base36 encoding errors."""
+
+
+def encode_base64(number):
```

Ruff (0.12.2)

Missing return type annotation for public function `encode_base64`

(ANN201)